src/guides/guides.11tydata.yaml

```
@@ -163,4 +163,28 @@ procedureStartup_after_power_off_disconnected:
               <li><b>Mobile</b>: Confirm the IP address detected by the app. For example <code>http://192.168.1.196:8123</code></li></li>
               <li><b>Desktop</b>: Visit <a href="http://homeassistant.local:8123" target="_blank">http://homeassistant.local:8123</a> to access the Home Assistant user interface.</li></li>
             </ul>         
+        </ul>
+procedureConnectTerminal:
+  title: Using the console on the HDMI display
+  steps:
+    - title: Connect an HDMI monitor
+      image: icons/hdmi-monitor.png
+      image-apply-filter: true
+      content: |
+        <ul>
+          <li>Connect an HDMI monitor.</li>
+        </ul>
+    - title: Connect a USB keyboard
+      image: icons/usb_keyboard.png
+      image-apply-filter: true
+      content: |
+        <ul>
+          <li>Connect a USB keyboard to Home Assistant Green.</li>
+        </ul>
+    - title: Check the console
+      image: home-assistant-cli.png
+      content: |
+        <ul>
+          <li>The console should open automatically and show up on the monitor.</li>
+          <li>In the Home Assistant command line, you can now enter a command.<ul><li>For example, if you are locked out, you can change your password: <br><code>`auth reset --username existing_user --password new_password`</code></li></ul></li>
```

No backticks required here IMHO.

```suggestion
          <li>In the Home Assistant command line, you can now enter a command.<ul><li>For example, if you are locked out, you can change your password: <br><code>auth reset --username existing_user --password new_password</code></li></ul></li>
```